src/components/ae-phraser/ae-phraser.vue

```diff
@@ -1,17 +1,64 @@
 <template>
-  <div class="ae-phraser">
-    <slot/>
+  <div class="ae-phraser" :class="{ error }">
+    <span
+      class="ae-phraser-error"
+      :class="{ active: Boolean(error) }"
+      v-if="error"
+    >
+      {{ error }}
+    </span>
+    <div :id="label" class="ae-phraser-container">
+      <slot/>
+    </div>
   </div>
 </template>
 <script>
 export default {
   name: 'ae-phraser',
-  props: {},
+  props: {
+    /**
+     * Sets the error state for the
+     * component, if a string is passed
+     * displays a message
+     */
+    error: [String, Boolean],
+  },
 };
 </script>
 <style lang="scss" scoped>
+@import '../../styles/variables/colors';
+@import '../../styles/placeholders/typography';
+
 .ae-phraser {
+  padding-left: 1rem;
+  border-left: 2px solid transparent;
+
+  &.error {
+    border-left-color: $color-error;
+  }
+}
+
+.ae-phraser-error {
+  @extend %face-sans-xs;
+
+  display: inline-block;
+  margin-bottom: 0.5rem;
+
+  &.active {
+    color: $color-error;
+  }
+}
+
+.ae-phraser-container {
   display: flex;
-  flex-direction: column;
+  align-items: flex-start;
+  flex-wrap: wrap;
+
+  /deep/ > .ae-badge {
```

Comment:
I'm not sure about the use of `/deep/`.
As Joel H. points out in [these comments](https://stackoverflow.com/questions/25609678/what-do-deep-and-shadow-mean-in-a-css-selector), Chrome has since deprecated the /deep/ combinator, and it gives a syntax error in IE.